CPP/data-structures/linkedlist.cpp

```diff
@@ -0,0 +1,60 @@
+#include<iostream>
+using namespace std;
+struct node{
```

Comment:
space after node

Reply:
s/node/Node

Reply:
make it generic, please

Reply:
Can you please elaborate what you expect with the generic notations?